scripts/collect-typos.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+pushd "$(git rev-parse --show-toplevel)" >/dev/null 
```

Comment:
can use scopes to avoid needing pushd/popd.

or use python3 or lua-nvim so that we don't have to think about tricks like this :)

Reply:
Yeah, I should have skipped bash and went straight to a "real" language.